subprojects/signing/src/main/java/org/gradle/plugins/signing/Signature.java

```diff
@@ -321,11 +321,6 @@ public void setFile(File file) {
      * @return The signature file. May be {@code null} if unknown at this time.
      */
     public File getFile() {
```

Comment:
I think we need to adjust the Javadoc for `getFile`, since it doesn't default anymore but is the default.

Reply:
Done in 62d13aecca547272b44f5797e6b5df49c53231b2.